pyspark/bigdl/models/lenet/utils.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.dataset import mnist
+from bigdl.dataset.transformer import *
+from bigdl.optim.optimizer import *
+
+
+def get_mnist(sc, data_type="train", location="/tmp/mnist"):
+    """
+    Get mnist dataset. We would download it automatically
+    if the data doesn't present at the specific location.
+
+    :param sc: SparkContext
+    :param data_type: "train" for training data and "test" for testing data
+    :param location: Location to store mnist dataset
+    :return: RDD of (features: ndarray, label: ndarray)
+    """
+    (images, labels) = mnist.read_data_sets(location, data_type)
+    images = sc.parallelize(images)
+    labels = sc.parallelize(labels + 1) # Target start from 1 in BigDL
+    record = images.zip(labels)
+    return record
+
+
+def process_mnist_rdd(sc, options):
```

Comment:
`get_mnist_rdd`?

Reply:
But in the above method `get_mnist` it already returns an RDD and this function does normalization and turn into Sample RDD...